pipes/WDL/workflows/train_16S_classifier.wdl

```diff
@@ -0,0 +1,26 @@
+version 1.0
+
+import "../tasks/tasks_16S_amplicon.wdl" as qiime
+
+workflow train_classifier_16S {
```

Comment:
```suggestion
workflow train_16S_classifier {
```
Having the workflow name match the wdl filename is kind of nice, but having the workflow name match the `name` of the workflow in `.dockstore.yml` is required (and currently that `name` is train_16S_classifier).